analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/FileMetadataAnalyzerTest.cs

```diff
@@ -38,46 +38,28 @@ public void Autogenerated(ProjectType projectType)
         {
             var autogeneratedFiles = new[]
             {
-                @"ExtraEmptyFile.g.cs", // This is added by Verifier, we will skip it when building the project
-                @"autogenerated_comment.cs",
-                @"autogenerated_comment2.cs",
-                @"class.designer.cs",
-                @"class.g.cs",
-                @"class.g.something.cs",
-                @"class.generated.cs",
-                @"class_generated.cs",
-                @"compiler_generated.cs",
-                @"compiler_generated_attr.cs",
-                @"debugger_non_user_code.cs",
-                @"debugger_non_user_code_attr.cs",
-                @"generated_code_attr.cs",
-                @"generated_code_attr2.cs",
-                @"TEMPORARYGENERATEDFILE_class.cs",
+                "ExtraEmptyFile.g.cs", // This is added by Verifier, we will skip it when building the project
+                "autogenerated_comment.cs",
+                "autogenerated_comment2.cs",
+                "class.designer.cs",
+                "class.g.cs",
+                "class.g.something.cs",
+                "class.generated.cs",
+                "class_generated.cs",
+                "compiler_generated.cs",
+                "compiler_generated_attr.cs",
+                "debugger_non_user_code.cs",
+                "debugger_non_user_code_attr.cs",
+                "generated_code_attr.cs",
+                "generated_code_attr2.cs",
+                "TEMPORARYGENERATEDFILE_class.cs",
```

Comment:
If you move this one after "ExtraEmptyFile.g.cs" they will be sorted alphabetically 😉

Reply:
after "ExtraEmptyFile.g.cs", not at the end. I usually use the editor to sort the selection lines and it's pretty neat.

Reply:
I'd keep `ExtraEmptyFile.g.cs` to semantically separate the scaffolding noise from the reset of the cases

Reply:
It was about moving 'TEMPORARYGENERATEDFILE_class.cs' not  `ExtraEmptyFile.g.cs`